DashAI/back/dataloaders/classes/dataloader.py

```diff
@@ -92,22 +92,10 @@ def extract_files(self, dataset_path: str, file: UploadFile) -> str:
 
     def _check_split_values(
         self,
-        dataset: DatasetDict,
         train_size: float,
         test_size: float,
         val_size: float,
-        stratify: bool = False,
```

Comment:
We are going to move the _split_dataset_ method from the **Dataloader** class to the **DashAIDataset** class, because the splitting of the dataset is done at experiment creation (linked to the **DashAIDataset**) instead of dataset creation (linked to the **Dataloader**). So we will add the option to _stratify_ in this new method.